lambdas/build-cri-oauth-request/src/main/java/uk/gov/di/ipv/core/buildcrioauthrequest/BuildCriOauthRequestHandler.java

```diff
@@ -302,6 +314,44 @@ private JWEObject signEncryptJar(
         return AuthorizationRequestHelper.createJweObject(rsaEncrypter, signedJWT);
     }
 
+    private EvidenceRequest getEvidenceRequestForF2F(List<SignedJWT> credentials)
+            throws UnknownEvidenceTypeException, ParseException {
+        Gpg45Scores gpg45Scores = gpg45ProfileEvaluator.buildScore(credentials);
+        List<List<Gpg45Scores.Evidence>> requiredEvidences =
+                gpg45ProfileEvaluator.calculateEvidencesRequiredToMeetAProfile(
+                        gpg45Scores, ACCEPTED_PROFILES);
+
+        if (requiredEvidences == null) {
+            return null;
+        }
+
+        OptionalInt minViableStrengthOpt =
+                requiredEvidences.stream()
+                        .filter(subList -> subList.size() == 1)
+                        .map(subList -> subList.get(0))
+                        .mapToInt(Gpg45Scores.Evidence::getStrength)
+                        .min();
+
+        return minViableStrengthOpt.isPresent()
+                ? new EvidenceRequest(minViableStrengthOpt.getAsInt())
+                : null;
```

Comment:
I think this is fine for now, since it replicates existing behaviour, but feels like this should actually be an error - it means we're about to send them to F2F but it's not possible for them to obtain a profile from it(!)

Reply:
Maybe we should add a warning log, which we can upgrade into an error if we confirm that it doesn't happen in production